PyHook.py

```diff
@@ -4,12 +4,32 @@
 
 
 from __future__ import print_function
+from argparse import ArgumentParser, RawTextHelpFormatter
 from multiprocessing.pool import ThreadPool
 from typing import List
 from time import sleep
 import psutil
 import re
 
+example_text = '''example:
+
+ python PyHook.py -p mstsc - Hook into mstsc
+ python PyHook.py -p mstsc, cmd - Hook into mstsc and cmd
+ python PyHook.py -p all - Hook all available processes
+ 
+available processes:
+
+ mstsc
+ cmd
+ mobaxterm
+ runas
+ explorer
+ psexec'''
+
+parser = ArgumentParser(epilog=example_text, formatter_class=RawTextHelpFormatter)
+parser.add_argument('-p', '--process', help='desired process to hook', nargs='+', required=True)
+args = parser.parse_args()
```

Comment:
Consider moving it to a function to avoid loading this code when importing the module.
Then using it in the main function.
```suggestion
def parse_args():
    parser = ArgumentParser(epilog=example_text, formatter_class=RawTextHelpFormatter)
    parser.add_argument('-p', '--process', help='desired process to hook', nargs='+', required=True)
    return parser.parse_args()
```
It's also better to pass the arguments "in-house" instead of using global variable.

Reply:
You said "Consider moving it to a function" I Didn't notice you defined any function in your suggestion

Reply:
line 29 in my suggestion

Reply:
![image](https://user-images.githubusercontent.com/43996999/132140310-7a2c317f-3139-4652-98c1-81e786b3cb93.jpeg)

I guess it's a bug in the GitHub mobile app 🤷‍♂️